### ISSUE TITLE
projects/core/howto/application.rst should be properly formatted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7640]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7640|
|Type|enhancement|
|Created|2014-09-16 11:14:30Z|
|Branch|https://github.com/twisted/twisted/tree/cleanup-rst-source-7640|

Cleaning up the rST source formatting

<details><summary>Searchable metadata</summary>

```
trac-id__7640 7640
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__ 
branch__branches_cleanup_rst_source_7640 branches/cleanup-rst-source-7640
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1410866070026521 1410866070026521
changetime__1411387955439565 1411387955439565
version__None None
owner__rwall rwall

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [43102]) Branching to cleanup-rst-source-7640.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl removed owner|
|-|-|

All cleaned up. Mainly whitespace, syntax, and moving one link for consistency.

**Comment by twisted-trac** [comment]
|[<img alt="derwolfe's avatar" src="https://avatars.githubusercontent.com/u/641357?s=50" width="50" height="50">](https://github.com/derwolfe)<a name="note_3"></a>|@derwolfe set owner to @hawkowl|
|-|-|

Replying to [hawkowl](#note_2):
> All cleaned up. Mainly whitespace, syntax, and moving one link for consistency.

The docs built well for me. I read through them and all sentences look to occupy a single line. You also removed all extraneous whitespace. 

I couldn't find anything that needs to be changed.  Please merge.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [43141]) Merge cleanup-rst-source-7640: application.rst should be properly formatted

Author: hawkowl
Reviewer: herrwolfe
Fixes: [#7640](https://github.com/twisted/twisted/issues/7640)
